pdns/dnsdistdist/docs/statistics.rst

```diff
@@ -99,6 +99,18 @@ latency-slow
 ------------
 Number of queries answered in more than 1 second.
 
+latency-sum
+-----------
+Total response time in milliseconds.
```

Comment:
It might make sense to mention the difference of this statistic in the time axis might say something about the change in latency. As latency usually is a gauge-type statistic over the last X seconds?

Reply:
I changed the documentation of this statistic a bit. I'm not sure how to really explain this metric as I'm not entirely sure what Prometheus will be using it for (other than it having to do with histograms).

Reply:
You can use it to calculate the average latency over a period of time: https://prometheus.io/docs/practices/histograms/#count-and-sum-of-observations

Reply:
Updated the documentation